app/src/main/java/com/kickstarter/ui/activities/ProjectActivity.kt

```diff
@@ -21,6 +21,7 @@ import androidx.fragment.app.FragmentManager
 import androidx.recyclerview.widget.LinearLayoutManager
 import com.google.firebase.crashlytics.FirebaseCrashlytics
```

Comment:
Project Activity works fine on my environment, Except the Campaign description and creator bio not loading.
- Like button: Ok
- Share button: ok
- Pledge Flow: Ok
- Manager pledge Flow: Ok
- Updates: Ok
- Comments: Ok